docker/prod/uwsgi/uwsgi.service

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ "${CONTAINER_NAME}" = "listenbrainz-web" ]
+then
+    exec uwsgi /etc/uwsgi/uwsgi.ini
+fi
+if [ "${CONTAINER_NAME}" = "listenbrainz-influx-writer" ]
+then
+    cd /code/listenbrainz
+    exec python influx-writer/influx-writer.py
+fi
+if [ "${CONTAINER_NAME}" = "listenbrainz-bigquery-writer" ]
+then
+    cd /code/listenbrainz
+    exec python bigquery-writer/bigquery-writer.py
+fi
+if [ "${CONTAINER_NAME}" = "listenbrainz-alpha-importer" ]
+then
+    cd /code/listenbrainz
+    exec python alpha_importer.py
+fi
+
+echo "init script has no clue which service to start. Set env var CONTAINER_NAME!"
```

Comment:
I'm not up to date on my bash - will this happen when the child process finished, or does the `exec` stop the rest of the script from running?
You could have a test for the environment variable at the top of this file instead, and have an explicit `exit 1` so that our init knows that it's failed (instead of exited) - http://stackoverflow.com/a/13864829 ?